2-mandatory/3-stocks.js

```diff
@@ -48,7 +65,19 @@ function getAveragePrices(closingPricesForAllStocks) {
     The price change value should be rounded to 2 decimal places, and should be a number (not a string)
 */
 function getPriceChanges(closingPricesForAllStocks) {
-    // TODO
+  // TODO
+  // initiate a new array priceChangeArr
+  let priceChangeArr = [];
+  // iterate input array
+  for (const closingPricesForIndividualCompany of closingPricesForAllStocks) {
+    // find price change by using index 0 and 4 to access values for Monday and Friday prices and take them away from each other
+    priceChange =
+      closingPricesForIndividualCompany[4] -
+      closingPricesForIndividualCompany[0];
+    // add priceChange value to array priceChangeArr. The value is rounded to 2 decimal using math.round and multiplied by 100 then divided by 100
```

Comment:
wow that very smart way to solve this one, and amazing work with the comments i should start writing more comments in my code as well.